tests/modules/aclcheck.c

```diff
@@ -193,13 +193,15 @@ int module_test_acl_category(RedisModuleCtx *ctx, RedisModuleString **argv, int
 int commandBlockCheck(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
     REDISMODULE_NOT_USED(argv);
     REDISMODULE_NOT_USED(argc);
-    int error = 0;
+    int response_ok = 0;
     int result = RedisModule_CreateCommand(ctx,"command.that.should.fail", module_test_acl_category, "", 0, 0, 0);
-    error |= (result == REDISMODULE_ERR);
+    response_ok |= (result == REDISMODULE_OK);
+
     RedisModuleCommand *parent = RedisModule_GetCommand(ctx,"block.commands.outside.onload");
     result = RedisModule_CreateSubcommand(parent,"subcommand.that.should.fail",module_test_acl_category,"",0,0,0);
-    error |= (result == REDISMODULE_ERR);
-    if (error) {
+    response_ok |= (result == REDISMODULE_OK);
+    
+    if (response_ok) {
         RedisModule_ReplyWithError(ctx, "NOPERM");
     } else {
         RedisModule_ReplyWithSimpleString(ctx, "OK");
```

Comment:
it looks a bit suspicious that we return "OK" when things failed.
maybe a comment saying that it validates that it's not possible to register these after loading, thus return an error if they succeed.

Reply:
or we can flip it and return back to using `assert_error {NOPERM}`.
i.e. my complaint was that the `|=` tested that at least one failed, rather than at least one succeeded.

Reply:
asserting the error makes more sense along with changing the name of the test function from `block.commands.outside.onload` to `create.commands.outside.onload`

Reply:
Yes, I will add a comment mentioning that and changing the error from `NOPERM` to `UNEXPECTEDOK` so it will make it more clear.